docs/source/tutorials/projects.rst

```diff
@@ -0,0 +1,1213 @@
+You can decide whether this new project also copies over the same users, datasets and models - which aren't copied over by default.
+
+The :meth:`copy_project() <.EncordClientProject>` method takes the following parameters, all of which are optional:
+
+* ``copy_datasets`` (default ``False``) when set to ``True``, the datasets from the original project will be copied over, and new tasks will be created from them
+* ``copy_collaborators`` (default ``False``) when set to ``True``, the collaborators from the original project will be copied over with their existing roles
+* ``copy_models`` (default ``False``) when set to ``True``, the models and their training data from the original project will be copied over
+
+The parameters above are set to ``False`` by default, meaning you do not need to include any of them if you
+do not want to copy that feature into your new project.
+
+:meth:`copy_project <.EncordClientProject.copy_project>` returns the ``<project_hash>`` of the new project.
+
+The following example showcases an example of copying a project.
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. literalinclude:: /code_examples/tutorials/projects/copying_a_project.py
+            :language: python
+
+    .. tab:: Example output
+
+        .. code-block:: python
+
+            "00000000-bbbb-cccc-eeee-ffffffffffff"  # the <project_hash>
+
+
+.. rubric:: Footnotes
+
+.. [#f1] If you do not copy over the collaborators, then the reviewer and label mapping settings won't be copied over.
+
+
+
+Listing existing projects
+=========================
+
+Via the :class:`.EncordUserClient`, you can query and list all the available projects of a given user.
+In the example below, a user authenticates with |company| and then fetches all projects available.
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. literalinclude:: /code_examples/tutorials/projects/listing_existing_projects.py
+            :language: python
+
+    .. tab:: Example output
+
+        .. code-block::
+
+            [
+                {
+                    'project': {
+                        'created_at': datetime.datetime(...),
+                        'description': 'Example description',
+                        'last_edited_at': datetime.datetime(...),
+                        'project_hash': '<project_hash>',
+                        'title': 'Example title'
+                    },
+                    'user_role': <ProjectUserRole.ADMIN: 0>
+                },
+                ...
+            ]
+
+
+.. note::
+
+    :meth:`.EncordUserClient.get_projects` has multiple optional arguments that allow you to query projects with specific characteristics.
+
+    For example, if you only want projects with titles starting with "Validation", you could use ``user_client.get_projects(title_like="Validation%")``.
+    Other keyword arguments such as :meth:`created_before  <.EncordUserClient.get_datasets>` or :meth:`edited_after <.EncordUserClient.get_datasets>` may also be of interest.
```

Comment:
Worth pointing them to the reference if they want ot know more

Reply:
In the built docs, all the mentions of `get_datasets` and arguments are links directly to where the reader would want to go. Should we still explicitly write it out in text you think?